pkg/cmd/cli/describe/printer_test.go

```diff
@@ -0,0 +1,69 @@
+package describe
+
+import (
+	"io/ioutil"
+	"reflect"
+	"strings"
+	"testing"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+	imageapi "github.com/openshift/origin/pkg/image/api"
+	projectapi "github.com/openshift/origin/pkg/project/api"
+)
+
+// PrinterCoverageExceptions is the list of API types that do NOT have corresponding printers
+// If you add something to this list, explain why it doesn't need validation.  waaaa is not a valid
+// reason.
+var PrinterCoverageExceptions = []reflect.Type{
+	reflect.TypeOf(&imageapi.DockerImage{}), // not a top level resource
+	reflect.TypeOf(&buildapi.BuildLog{}),    // just a marker type
+}
+
+// MissingPrinterCoverageExceptions is the list of types that were missing printer methods when I started
+// You should never add to this list
+var MissingPrinterCoverageExceptions = []reflect.Type{
```

Comment:
Some of these may not seem like candidates, but remember that `resource.Builder` covers more types than just straight API server GETs.